inferentia/README.md

```diff
@@ -87,6 +100,18 @@ you can run:
 source /home/ubuntu/python_backend/inferentia/scripts/setup.sh -v 3.6
 ```
 Please use the `-h` or `--help` options to learn about more configurable options.
+## Compile Inferentia Model

```

Comment:
This can be removed as compiling the model for inferentia is a part of section [Setting up the Inferentia model](setting-up-the-inferentia-model). See below. For TF, we don't have `*.pt` or `*.pb` models, but have TF savedmodel. If you have something to add, then you can add in that section.